src/mini_s3.erl

```diff
@@ -459,7 +483,7 @@ s3_url(Method, BucketName, Key, Lifetime, RawHeaders,
     Path = lists:flatten([$/, BucketName, $/ , Key]),
     CanonicalizedResource = ms3_http:url_encode_loose(Path),
 
-    {_StringToSign, Signature} = make_signed_url_authorization(SecretKey, Method,
+    {_StringToSign, Signature} = mini_s3:make_signed_url_authorization(SecretKey, Method,
```

Comment:
Just curious, what's the reasoning for making this call fully qualified with `mini_s3:...` now?